app/views/layouts/_footer.html.erb

```diff
@@ -42,7 +42,7 @@
     </div>
     <div class="footer-copyright-language-row">
       <div class="footer-copyright">
-        &copy; 2011&ndash;2020 <%= _("openSUSE contributors") %>
+        &copy; 2011&ndash;<%= Time.new.year %> <%= _("openSUSE contributors") %>
```

Comment:
I can imagine that this should be cached and not calculated for every request.

I don't know how expensive it is for Ruby to calculate the current year vs accessing a cache, just a thought

Reply:
I don't know how expensive it is, either, let alone if it's worth it to add the whole caching song and dance. I'm far too non-well-versed in Ruby to do better than an educated guess, which is "unless Ruby is really _horrendously slow_, it should be a non-issue".

Reply:
I think you're right, let's leave it like that.